src/rng/backends/mklcpu/mrg32k3a.cpp

```diff
@@ -33,6 +33,8 @@ namespace mkl {
 namespace rng {
 namespace mklcpu {
 
+using namespace cl;
```

Comment:
Why is this needed?

Reply:
It is needed since for example in line 62 `sycl::buffer` is not qualified with `cl::`

Reply:
Okay.. that's arguably a bug in oneMKL then. I don't think the spec allows using `::sycl` directly when including `CL/sycl.hpp` and not `sycl/sycl.hpp`. But it's not our problem then :)